src/vaadin-grid-event-context-mixin.html

```diff
@@ -30,6 +30,9 @@
      * This means mostly mouse and keyboard events. If such a grid part is not found in the path, an empty object is returned.
      * This may be the case eg. if the event is fired on the `<vaadin-grid>` element and not any deeper in the DOM, or if
      * the event targets the empty part of the grid body.
+     *
+     * @param {Event} event
+     * @return {GridEventContext}
```

Comment:
This needs a non-null `Event`. Also this never returns null.
```suggestion
     * @param {!Event} event
     * @return {!GridEventContext}
```
But this may return an empty object `{}` but `GridEventContext` interface requires at least `section`.

So imo the interface should be updated to make also `section` optional. Or alternatively this method implementation may be changed to return `null` instead of `{}` and then use `@return {GridEventContext}` but that would change the behaviour and I didn't check if that would require other changes. So would be safer for now to make the interface allow empty object? Or make the return type something that means "GridEventContext or empty object".

Reply:
Changed to `{GridEventContext | object}`.